src/polar/Pie.tsx

```diff
@@ -59,7 +59,7 @@ interface PieDef {
   /** The inner radius of sectors */
   innerRadius?: number | string;
   /** The outer radius of sectors */
-  outerRadius?: number | string;
+  outerRadius?: number | string | ((element: any) => number);
```

Comment:
`element` implies "html element" or "react element". Can we name this something related to data? 

`dataItem`, `dataPoint`, even just `item` would be ok

Reply:
made the change

Reply:
I may be missing something but I don't see the changes. Can you try to push again?

Reply:
done